plugins/test/dynamic_test.cc

```diff
@@ -394,6 +550,16 @@ void DynamicTest::BenchHttpHandlers(benchmark::State& state) {
     std::vector<std::string> request_body_chunks_copies = *request_body_chunks;
     std::vector<std::string> response_body_chunks_copies =
         *response_body_chunks;
+
+    // Advance a fraction of background streams, if present.
```

Comment:
This is happening while timing is paused. Is the intention such that background streams will load the VM, but not impact the CPU or wall time measurements of the test cases?

I suppose that's necessary, since otherwise the benchmark's values wouldn't always converge and wouldn't be a strict measurement of the test case.

Could you add a comment to the new proto fields to explain the impact they have on measurements? I.e. something like "The cpu time of the background streams is not measured in the microbenchmark. Rather, the background streams allow us to test the cost of a single stream while the plugin is loaded with other streams, which may catch memory usage issues or performance issues with algorithmic inefficiencies with large number of in-flight requests"

Reply:
> This is happening while timing is paused. Is the intention such that background streams will load the VM, but not impact the CPU or wall time measurements of the test cases?

Yes, exactly.

> Could you add a comment to the new proto fields to explain the impact they have on measurements?

Sure--added the text you suggested. Thanks!